jaeger_client/sampler.py

```diff
@@ -45,7 +49,7 @@
 SAMPLER_TYPE_TAG_KEY = 'sampler.type'
 SAMPLER_PARAM_TAG_KEY = 'sampler.param'
 DEFAULT_SAMPLING_PROBABILITY = 0.001
-DEFAULT_LOWER_BOUND = 1.0 / (10.0 * 60.0)  # sample once every 10 minutes
+DEFAULT_LOWER_BOUND = old_div(1.0, (10.0 * 60.0))  # sample once every 10 minutes
```

Comment:
What's the reason for this change? Doesn't Python3 work with `1.0 / (10.0 * 60.0)`? 

I'd like to have comments in the code for non-trivial decisions.